playground/OpenAIEndToEnd/OpenAIEndToEnd.WebStory/Components/Pages/UseIChatClient.razor

```diff
@@ -0,0 +1,37 @@
+﻿@page "/useichatclient"
```

Comment:
Can this just go in the existing `Home.razor`? I'm not sure why we'd have/need both.

Reply:
`Home.razor` is for using `OpenAIChatClient`. This one is for using `IChatClient`.

Reply:
I think we should just move this logic into Home.razor. I don't see a need to have both.

Also, I pulled the changes locally and this page doesn't appear to work:

```
2024-11-26T12:32:38       System.InvalidOperationException: An IChatClient could not be configured. Ensure a 'Deployment' or 'Model' value is provided in 'ConnectionStrings:openai', or specify a 'Deployment' in the 'Aspire:Azure:AI:OpenAI' configuration section, or specify a 'deploymentName' in the call.
```

I assume we need to change the AppHost

Reply:
> I assume we need to change the AppHost

My PR is definitely changing how these settings are set in the app host.

Reply:
> Also, I pulled the changes locally and this page doesn't appear to work:

Thanks. I've updated the E2E code so that it uses the same technique to get the deployment name in the new E2E page as it did in the old one, so it should work consistently for the two.

> My PR is definitely changing how these settings are set in the app host.

Makes sense!